migrations/000002.php

```diff
@@ -0,0 +1,7 @@
+<?php
+// This migration adds an "enabled" column to the payment_methods table and sets all values to 1.
+// It allows the user to disable payment methods without deleting them.
+
```

Comment:
I know the migrator stores which migrations were made, but I would feel better if we checked for the existence of the column before applying the changes below.

Reply:
I've updated the migration now.

I think also perhaps the migrations could be named with the date in ISO8601 format or unix timestamp or something instead of just a sequence.
This could prevent clashes with multiple PRs coming in at once.